tests/test_views.py

```diff
@@ -360,12 +361,12 @@ def test_get_with_identifier_response_content(self, rf):
         agent = factories.AgentFactory.create()
         request = rf.get('/')
         response = views.app_agent(request, agent.agent_identifier)
-        assert agent.agent_identifier in response.content
+        assert agent.agent_identifier in response.content.decode('utf-8')
```

Comment:
I can't help but think that maybe instead of using so many decodes in the tests (not only in this file but the other tests as well), we should just change our string literals to bytes literals?

Reply:
I think if that's all we're using particular strings for (checking directly against response content), then yeah, let's change it there and keep a tiny bit of complexity out of the tests. Additionally, there are some string literals defined directly in the tests that can be changed to avoid the extra decodes.

Reply:
Backtracking this: let's actually leave the factories as they are, and only make changes to the really simple assertions where we're testing that an in-line string literal is in the response content.